helpers/isValidId.js

```diff
@@ -0,0 +1,13 @@
+const { isValidObjectId } = require("mongoose");
+const HttpError = require("./HttpError");
+
+const isValidId = (req, _, next) => {
+  if (!isValidObjectId(req.params.id)) {
+    next(new HttpError(404));
+    return;
```

Comment:
```suggestion
    return next(new HttpError(404));
```
Tyle że bardziej eleganckie. ;)